CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java

```diff
@@ -25,6 +25,12 @@ protected void onCreate(Bundle savedInstanceState) {
 
         FirebaseFirestore db;
 
+        if (!((GroupProject) this.getApplication()).isSignedIn()) {
+            Intent intent = new Intent(this, SignInActivity.class);
+            ((GroupProject) this.getApplication()).setSignedIn(true);
```

Comment:
Set signedIn to true after actually signing in.

Reply:
The implementation of signedIn currently only affects anything on initial app start up. I'll change it incase the functionality is changed in the future though :)